rules/windows/privilege_escalation_printspooler_service_suspicious_file.toml

```diff
@@ -74,3 +103,9 @@ id = "TA0004"
 name = "Privilege Escalation"
 reference = "https://attack.mitre.org/tactics/TA0004/"
 
+[rule.new_terms]
+field = "new_terms_fields"
+value = ["host.id", "file.path"]
```

Comment:
Wouldn't it be beneficial to remove host.id here, as I can assume this rule being quite noisy, and having other hosts to reduce noise would be useful?

Reply:
I'll try it with the host first and make it broader if needed